systemtest/src/test/java/io/strimzi/systemtest/security/NetworkPoliciesST.java

```diff
@@ -189,67 +143,99 @@ void testNetworkPoliciesWithTlsListener(ExtensionContext extensionContext) {
                             .withNetworkPolicyPeers(
                                 new NetworkPolicyPeerBuilder()
                                     .withNewPodSelector()
-                                        .addToMatchLabels("app", testStorage.getProducerName())
+                                    .addToMatchLabels("app", producerNameAccessedTls)
                                     .endPodSelector()
                                     .build(),
                                 new NetworkPolicyPeerBuilder()
                                     .withNewPodSelector()
-                                        .addToMatchLabels("app", testStorage.getConsumerName())
+                                    .addToMatchLabels("app", consumerNameAccessedTls)
                                     .endPodSelector()
                                     .build()
                             )
-                            .build())
+                            .build()
+                    )
                 .endKafka()
+                .withNewKafkaExporter()
+                .endKafkaExporter()
             .endSpec()
             .build(),
-            KafkaTopicTemplates.topic(testStorage.getClusterName(), topic0, testStorage.getNamespaceName()).build(),
-            KafkaTopicTemplates.topic(testStorage.getClusterName(), topic1, testStorage.getNamespaceName()).build(),
-            KafkaUserTemplates.scramShaUser(testStorage).build()
+            ScraperTemplates.scraperPod(testStorage.getNamespaceName(), testStorage.getScraperName()).build(),
+            KafkaUserTemplates.scramShaUser(testStorage).build(),
+            KafkaTopicTemplates.topic(testStorage.getClusterName(), topicNameAccessedPlain, testStorage.getNamespaceName()).build(),
+            KafkaTopicTemplates.topic(testStorage.getClusterName(), topicNameAccessedTls, testStorage.getNamespaceName()).build(),
+            KafkaTopicTemplates.topic(testStorage.getClusterName(), topicNameDeniedPlain, testStorage.getNamespaceName()).build(),
+            KafkaTopicTemplates.topic(testStorage.getClusterName(), topicNameDeniedTls, testStorage.getNamespaceName()).build()
         );
 
-        LOGGER.info("Verifying that producer/consumer: {}/{} are able to exchange messages", testStorage.getProducerName(), testStorage.getConsumerName());
+        LOGGER.info("Initialize producers and consumers with access to the Kafka using plain and tls listeners");
+        KafkaClients kafkaClientsWithAccessPlain = ClientUtils.getDummyClientBuilder(testStorage)
+            .withBootstrapAddress(KafkaResources.plainBootstrapAddress(testStorage.getClusterName()))
```

Comment:
You have `plainBootstrapAddress` already in the clients method, so you don't need to specify it here (just a nit)

Reply:
sure, makes sense 👍